packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -243,6 +260,10 @@ export const Datepicker: ComponentWithAs<'div', DatepickerProps> &
         e.preventDefault();
       },
     },
+    mapPropsToBehavior: () => ({
+      'aria-invalid': a11yProps['aria-invalid'],
+      'aria-labelledby': a11yProps['aria-labelledby'],
```

Comment:
```suggestion
      'aria-invalid': props['aria-invalid'],
      'aria-labelledby': props['aria-labelledby'],
```

We have the same thing in `Image` component:
https://github.com/microsoft/fluentui/blob/46055c7b84750bda3d609ab8bf58bb53e4e952cd/packages/fluentui/react-northstar/src/components/Image/Image.tsx#L21

I suggest to move `aria-labelledby` & `aria-invalid` to `DatepickerProps` instead and add a comment that they will be applied to an another slot. Once they will be under `Datepicker.handledProps` they will not be longer a part of `unhandledProps` and will not need changes in `partitionHTMLProps`.